packages/medusa/src/types/customer-groups.ts

```diff
@@ -7,4 +7,7 @@ export class FilterableCustomerGroupProps {
   @ValidateNested()
```

Comment:
make sure to include the additional fields specified in the request params above :)

Reply:
You may find inspiration from `list-orders` where the `FilterableOrderProps` are extended to form `GetAdminOrdersParams`.